lib/platform/gitlab/index.ts

```diff
@@ -479,7 +479,7 @@ export async function getPr(iid: number): Promise<Pr> {
     const branch = (
       await gitlabApi.getJson<{ commit: { author_email: string } }>(branchUrl)
     ).body;
-    const branchCommitEmail = branch.commit.author_email;
+    const branchCommitEmail = branch?.commit?.author_email ?? null;
```

Comment:
Looks like we have a bad test? We can ignore that for now because #6697 will remove this anyways

Reply:
Yeah, it's been like that for so long, I can live with it until we merge that PR.